tool/gravity/cli/operation.go

```diff
@@ -163,18 +163,37 @@ func completeOperationPlan(localEnv *localenv.LocalEnvironment, environ LocalEnv
 	}
 	switch op.Type {
 	case ops.OperationInstall:
-		return completeInstallPlan(localEnv, op)
+		err = completeInstallPlan(localEnv, op)
 	case ops.OperationExpand:
-		return completeJoinPlan(localEnv, op)
+		err = completeJoinPlan(localEnv, op)
 	case ops.OperationUpdate:
-		return completeUpdatePlan(localEnv, environ, *op)
+		err = completeUpdatePlan(localEnv, environ, *op)
 	case ops.OperationUpdateRuntimeEnviron:
-		return completeEnvironPlan(localEnv, environ, *op)
+		err = completeEnvironPlan(localEnv, environ, *op)
 	case ops.OperationUpdateConfig:
-		return completeConfigPlan(localEnv, environ, *op)
+		err = completeConfigPlan(localEnv, environ, *op)
 	default:
 		return trace.BadParameter("operation type %q does not support plan completion", op.Type)
 	}
+	if trace.IsNotFound(err) {
+		return completeClusterOperationPlan(localEnv, *op)
```

Comment:
Does this address part 2 of https://github.com/gravitational/gravity.e/issues/3215 by chance? I.e. can we "complete" arbitrary operation using a CLI command now?

Reply:
Yes.